content/en/synthetics/mobile_app_testing/_index.md

```diff
@@ -26,32 +26,141 @@ cascade:
 <div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
 {{< /site-region >}}
 
+## Overview
 
-Mobile Application Testing allows you to test and monitor key business flows for Android and iOS applications using real devices. Datadog runs these tests on real devices to provide a realistic, step-by-step representation of key application workflows, screenshots of each step, and detailed pass or fail results so your team can quickly visualize what went wrong.
+Mobile Application Testing allows you to test and monitor key business flows for Android and iOS applications using real devices. 
+Datadog runs these tests on real devices to provide a realistic, step-by-step representation of key application workflows, screenshots of each step, and detailed pass or fail results so your team can quickly visualize what went wrong.
+Mobile app tests can run on a schedule, on-demand, or directly within your [CI/CD pipelines][1].
 
-## Record mobile app tests
+You can create mobile app tests in Datadog by navigating to [**Digital Experience** > **New Test**][12] and selecting **Mobile Application Test**.
 
-[Record mobile app tests][1] for Android and iOS using the no-code test recorder, which makes it easy to create robust and comprehensive tests.
+{{< img src="mobile_app_testing/new_test_2.png" alt="Create a Synthetic Mobile Test" style="width:50%;">}}
 
-{{< img src="mobile_app_testing/mobile-test-screenshots.mp4" alt="Examples of the recording workflow for a Synthetic Mobile Test" video="true" >}}
+### Flakiness 
 
-## Run tests on CI or manually run tests
+Flakiness is a pain point in end-to-end testing because tests occasionally fail. When a frontend team implements a change, an identifier in your test may alert on it instead of an actual application issue.
```

Comment:
```suggestion
Flakiness is a pain point in end-to-end testing. Test failures are occasionally caused by valid frontend code changes that impact an identifier, not by an actual application issue.
```
Rewording just because the original felt a little awkward -- identifiers don't alert.